### PR TITLE
replace missing MDI icons with existing ones

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

- replace Lobby and GridView icons with analogs from library
- move the rest icons from `components` to `assets`

## 🖌️ UI Checklist

Check locked Lobby room as a guest and TopBar in a call

| 🏚️ Before | 🏡 After |
|------------|----------|
image | image
image | image
image | image
image | image
image | image

### 🏁 Checklist

- [x] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences

### CODE REVIEW

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: src/components/LobbyScreen.vue [hunk 1]
```diff
@@ -49,23 +49,24 @@
...
-import Lobby from './missingMaterialDesignIcons/Lobby.vue'
 import SetGuestUsername from './SetGuestUsername.vue'
 
 export default {
 
 	name: 'LobbyScreen',
 
 	components: {
-		SetGuestUsername,
-		NcRichText,
```
[the remark this answers is not in the document]
Comment (DorraJaouad) [reply src/components/LobbyScreen.vue:66]: It is just reordered

**Comment by DorraJaouad** [comment]
Suggestions for icons 

CancelPresentation -> https://pictogrammers.com/library/mdi/icon/monitor-off/
CategoryMonitoring -> [pulse](https://pictogrammers.com/library/mdi/icon/pulse/) OR [clip board pulse outline](https://pictogrammers.com/library/mdi/icon/clipboard-pulse-outline/)

**Comment by nickvergessen** [comment]
pulse should be fine and I guess also monitor off?

**Comment by Antreesy** [comment]
Replaced PresentToAll with MonitorShare for consistency
_more monitors..._

**Review by DorraJaouad — APPROVED** [review]
Looks good 🖼️

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**